tests/test_create_inconsistent_dimensionality.py

```diff
@@ -20,19 +21,34 @@
+]
+
+
 wkt_cases = [
     ('LINESTRING (1 1 1, 2 2)', 'LINESTRING (1.0000000000000000 1.0000000000000000, 2.0000000000000000 2.0000000000000000)'),
     ('POLYGON ((0 0 0, 1 0 0, 1 1, 0 1 0, 0 0 0))', 'POLYGON ((0.0000000000000000 0.0000000000000000, 1.0000000000000000 0.0000000000000000, 1.0000000000000000 1.0000000000000000, 0.0000000000000000 1.0000000000000000, 0.0000000000000000 0.0000000000000000))')
 ]
 
 
+
 @mark.parametrize('geojson', geojson_cases)
 def test_create_from_geojson(geojson):
     with pytest.raises(ValueError) as exc:
         wkt = asShape(geojson).wkt
         assert exc.match("Inconsistent coordinate dimensionality")
 
 
+@mark.parametrize('constructor, args', direct_cases)
+def test_create_directly(constructor, args):
+    with pytest.raises(ValueError) as exc:
+        geom = constructor(*args)
+        assert exc.match("Inconsistent coordinate dimensionality")
```

Comment:
This assert needs to come out of the `with` block, otherwise it will never be reached.

Reply:
Duh. Thanks.